automation/release.sh

```diff
@@ -71,6 +71,23 @@ function update_github_release() {
         _out/cmd/cniplugins/*
 }
 
+function upload_github_source_tarball_signature() {
```

Comment:
`update_github_release` this could probably be merged to and upload the signature right away. Am I missing anything?

Reply:
I was using a separate function in order to be able to test it apart from the remaining release creation. Do you see that as a problem?

Reply:
This way we might be able to leverage it for creating signature files for existing releases.

Reply:
^^ @xpivarc WDYT?

Reply:
I would still refactor a signing function and integrate it to the normal flow, we can have a function for existing releases that verifies that the release is signed and it can sign it if its not already

Reply:
Ah, now I get your point: you want idempotency. Thanks, makes sense!